src/EscolaLmsTemplatesPdfServiceProvider.php

```diff
@@ -32,6 +32,14 @@ public function register()
 
         $this->app->register(AuthServiceProvider::class);
         $this->app->register(UserTemplateServiceProvider::class);
+
+        if (class_exists(\EscolaLms\Settings\Facades\AdministrableConfig::class)) {
+            $this->mergeConfigFrom(
+                __DIR__ . '/config.php',
+                self::CONFIG_KEY
+            );
```

Comment:
```suggestion
```

a few lines above ([27](https://github.com/EscolaLMS/Templates-PDF/blob/0640d4afed6138372f2f694af173f534dd44902f/src/EscolaLmsTemplatesPdfServiceProvider.php#L27)) is the same code fragment